src/Command/Cli/DeleteOldGiftCardsCommand.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusGiftCardPlugin\Command\Cli;
+
+use Doctrine\Persistence\ObjectManager;
+use Setono\SyliusGiftCardPlugin\Repository\GiftCardRepositoryInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class DeleteOldGiftCardsCommand extends Command
+{
+    protected static $defaultName = 'setono:sylius-gift-card:delete-old-gift-cards';
+
+    private GiftCardRepositoryInterface $giftCardRepository;
+
+    private ObjectManager $giftCardManager;
+
+    public function __construct(
+        GiftCardRepositoryInterface $giftCardRepository,
+        ObjectManager $giftCardManager,
+        ?string $name = null
```

Comment:
```suggestion
```

Why do you need the name?

Reply:
To call parent constructor

Reply:
Yeah, but it's null by default, so you don't need it in your constructor :)